lib/child_process.js

```diff
@@ -70,10 +70,10 @@ exports._forkChild = function(fd) {
   p.open(fd);
   p.unref();
   const control = setupChannel(process, p);
-  process.on('newListener', function(name) {
+  process.on('newListener', function newListenerCallback(name) {
```

Comment:
Feel free to ignore, but perhaps name this `onNewListener`. Same with `onRemoveListener`.